Source/WebKit/NetworkProcess/storage/NetworkStorageManager.cpp

```diff
@@ -222,6 +222,7 @@ NetworkStorageManager::~NetworkStorageManager()
 {
     ASSERT(RunLoop::isMain());
     ASSERT(m_closed);
+    m_queue->dispatch([storageManger = WTFMove(m_sharedServiceWorkerStorageManager)] { });
```

Comment:
Good catch! This should be moved to `NetworkStorageManager::close`, inside the task dispatched to m_queue.

Reply:
Good point. Will fix. However, doing `m_sharedServiceWorkerStorageManager = nullptr;` in the dispatched task causes another problem if I put a break point in `SWRegistrationDatabase::close`. I have to use `WTFMove(m_sharedServiceWorkerStorageManager)` in the lambda capture.

Reply:
Fixed.